gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 #Sat May 22 22:20:14 ART 2021
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-bin.zip
```

Comment:
Can be updated to 7.4.1 by executing `./gradlew wrapper --gradle-version=7.4.1`.

Reply:
Yep.

Reply:
May something with `robolectric`, no idea too.